src/context/auth-context.tsx

```diff
@@ -97,7 +102,7 @@ export const AuthDataProvider = ({ children }: AuthDataProviderProps) => {
     const value = useMemo(
         () => ({
             activeLoginid,
-            isAuthorizing: isPending,
```

Comment:
the issue with using isPending was that the initial value was false instead of true because when the react-query's status is `idle`, `isPending` will be false. but this is not the behavior that we wanted. we want `isAuthorizing` to be True at first and once the authorization is done it changes to `false`.

Reply:
@shayan-deriv, did you test `authorized subscriptions` after this change? do they still work as expected?